static/app/utils/onDemandMetrics/index.tsx

```diff
@@ -110,25 +131,72 @@ export const extrapolatedAreaStyle = {
   opacity: 1.0,
 };
 
-export function hasOnDemandMetricAlertFeature(organization: Organization) {
-  return organization.features.includes('on-demand-metrics-extraction');
-}
-
-export function hasOnDemandMetricWidgetFeature(organization: Organization) {
+export function OnDemandWarningIcon({
+  msg,
+  color = 'gray300',
+}: {
+  msg: React.ReactNode;
+  color?: Color;
+}) {
   return (
-    organization.features.includes('on-demand-metrics-extraction') &&
-    organization.features.includes('on-demand-metrics-extraction-experimental')
+    <Tooltip title={msg}>
+      <HoverableIconWarning color={color} />
+    </Tooltip>
   );
 }
```

Comment:
Do we usually put components into the utils?
Should we rather have an `onDemandMetrics` folder inside `components`?

Reply:
Moved components to `static/app/components/alerts/onDemandMetricAlert.tsx`